### PR TITLE
Ruby on Rails capstone project - Budget app

### PR DESCRIPTION
**Pull request by LuisSalas94**
**Hello Microverse Team**

_In this branch I :_ 
- Used ruby gems as a software package system.
- Installed Ruby on Rails framework.
- Understood Rails RESTful design and router.
- Used controllers to handle requests and render empty views.
- Used params from browser requests in a safe way.
- Used preprocessed HTML files with embedded Ruby code.
- Used layouts and templates for shared content.
- Used database migration files to maintain database schema.
- Used validations for models.
- Secured app from n+1 problems.
- Understood what ORM is.
- Wrote SQL queries with ActiveRecord.
- Set up associations between models.
- Build a web app that requires the user to log in.
- Used devise gem for authentication.
- Limited access to web app resources based on authorization rules.
- Analyzed in writing why you have made a coding choice using one structure over another.

**Happy Coding**

### CODE REVIEW

File: spec/views/group_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    it 'show show add category button' do
+      expect(page).to have_content('ADD CATEGORY')
+    end
+
+    it 'should take to new group form' do
+      click_button 'Add Category'
+      expect(page).to have_content('NEW CATEGORY')
+    end
+  end
+end
```
Comment (DeliceLydia) [inline spec/views/group_index_spec.rb]: - [ ] Please fix the test that is failing
fails

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @LuisSalas94, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there! Major highlights for your project:
- App is working as expected
- No linter errors


## Suggested changes
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/views/group_index_spec.rb (hunk 1 above)
Comment (LuisSalas94) [reply spec/views/group_index_spec.rb:25]: Hello @DeliceLydia 
I just check it, It's working on my computer.

**Comment by LuisSalas94** [comment]
# Required Changes have been implemented 
- Test fixed
final-test

**Review by andersonlebon — APPROVED** [review]
## Hi @LuisSalas94 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**